cmd/util/ledger/migrations/change_contract_code_migration.go

```diff
@@ -350,3 +363,20 @@ func mustHexAddress(hexAddress string) common.Address {
 	}
 	return address
 }
+
+// NewlyAddedSystemContracts returns the list of newly added system contracts.
+// These needs to be added to the existing payloads.
+func NewlyAddedSystemContracts(chainID flow.ChainID) []*ledger.Payload {
+	systemContracts := systemcontracts.SystemContractsForChain(chainID)
+	serviceAccount := systemContracts.FlowServiceAccount.Address
+
+	return []*ledger.Payload{
+		// `Burner` contract
+		ledger.NewPayload(
+			convert.RegisterIDToLedgerKey(
+				flow.ContractRegisterID(serviceAccount, "Burner"),
+			),
+			ftContracts.Burner(),
+		),
+	}
+}
```

Comment:
This is only used for test at the moment. We would need to properly 'deploy' this contract somehow